gramps_webapi/api/resources/tasks.py

```diff
@@ -39,13 +39,14 @@ def get(self, task_id: str):
 
         def serialize_or_str(obj):
             try:
-                return json.dumps(obj)
+                return object_to_string(obj) # json.dumps(obj)
```

Comment:
This module is quite sloppy because it doesn't use any type hints - in this function, `obj` is not a Gramps object, so there is no need to change it, we can leave `json.dumps`.

Reply:
All of the json methods have been replaced with optimized versions. If obj is actually a dict, then you should use `dict_to_string` (which uses orjson in this case, and is much faster). But anyway, we want to hide the details in these methods as they are all optimized and will always be the best methods of conversion, even if the internal representation changes going forward.